### ISSUE TITLE
Current State of the Project

### ISSUE THREAD

**Issue by josiahseaman** [janpaul123/paperprograms#80]
Hello,
Please, what is the current state of the project?  I've been really excited about this program and have been telling people about it for the last two years.  I was looking at the Fork Network and vedran and cesmoak have both done substantial work that hasn't been merged into janpaul123's repo.  Does anyone else have a desire to see paper programs continue?  I've read #32.  I think Paper Programs would be a great education platform for kids.  I'd like to see it in our local library or in houses with homeschoolers.  I'm planning on visiting Dynamicland eventually, but a project that can be installed is a lot more accessible to people that can't afford the plane ticket.  2018 was two years ago now and I haven't seen any new content up on their website.  

So is there anyone else who would like to continue Paper Programs?

**Comment by janpaul123** [comment]
Yeah I haven't been working on this. That said, I'd be happy to take any good PRs if you do want to work on this.

I'm also just very curious to see how this sort of technology will take shape in the next few years. Dynamicland has been making tremendous progress, and we're also starting to see niche commercial devices coming out like [this](http://m.puppyrobot.com/en/puppycube) and [this](https://www.sonymobile.com/us/products/smart-products/xperia-touch/) that start using some of the affordances that we've been playing with here (though nothing like a powerful programming model like Realtalk). I just don't have too much time to devote to this right now unfortunately!

**Comment by josiahseaman** [comment]
@vedran or @cesmoak, do you have code you would like to PR into the main repo?  Neither of you have issues enabled, so discussion has to happen on Jan Paul's repo.

**Comment by vedran** [comment]
Sorry for the super late reply, I stumbled onto this issue coincidentally and must have missed notifications. I don't think I have anything that is safe to merge into the general repo though.

**Comment by josiahseaman** [comment]
Now that we've got everyone in quarantine for the foreseeable future, anyone feel the need to revive this project?  I could see this being very popular educational tool now that everyone is homeschooling their kids.

**Comment by brettfiedler** [comment]
Verified with JP before the plug. For those interested in what Paper Programs does and who may stumble across this issue:

[Paper Playground](https://github.com/phetsims/paper-land) is a "spiritual continuation" of JP's project here. We're aiming to capture the embodied and collaborative coding aspects of Paper Programs while focusing on specific outcomes of facilitating multimodal web-interactive codesign with technical and non-technical users. So far, it continues to retain all of the features of Paper Programs and adds quite a bit more! Use and contribution is highly encouraged. Everything remains openly licensed and community-oriented.
